City-Bureau/city-scrapers-bismarck
# 🏗️ Build spider: Morton County Commission

## What's this PR do?
Adds a spider to scrape meetings from the website of Morton County Commission. The new spider is called `bisnd_mcc`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl bisnd_mcc -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here]().

## Are there any smells or added technical debt to note?
- This agency provides great links to agendas and minutes but otherwise doesn't provide great meeting info (eg. meeting title or meeting start time). Certain key details are hardcoded.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206619009375822
